src/Compilers/Test/Resources/Core/NetFX/Minimal/mincorlib.cs

```diff
@@ -2,8 +2,8 @@
 
 namespace System
 {
-	public class Object {}
-    public struct Byte {}
+    public class Object { }
```

Comment:
That said we should certainly add targeted tests with their private corlib that's missing some of these types to validate we report an error. #9463 tracks that for the types added here.